src/App.js

```diff
@@ -14,11 +16,97 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGoods: [],
+  }
+
+  componentDidMount() {
+    this.setState({ selectedGoods: ['Jam'] });
+  }
+
+  setTitleText = () => {
```

Comment:
```suggestion
  getTitleText = () => {
```
When we get some text from the function, it's usually starts with `get`